packages/ui-form-field/src/FormFieldLayout/styles.ts

```diff
@@ -158,20 +159,16 @@ const generateStyle = (
     },
     formFieldLabel: {
       label: 'formFieldLayout__label',
-      ...(hasVisibleLabel && {
-        ...labelStyles,
-        // NOTE: needs separate groups for `:is()` and `:-webkit-any()` because of css selector group validation (see https://www.w3.org/TR/selectors-3/#grouping)
-        '&:is(label)': labelStyles,
-        '&:-webkit-any(label)': labelStyles
-      })
+      ...labelStyles,
```

Comment:
hasVisibleLabel is not needed here because this check already happens in `index.tsx`